io_scene_niftools/modules/nif_export/property/material/__init__.py

```diff
@@ -100,7 +100,7 @@ def export_material_property(self, b_mat, flags=0x0001):
         n_mat_prop.emissive_color.b = emissive.b
 
         # gloss mat 'Hardness' scrollbar in Blender, takes values between 1 and 511 (MW -> 0.0 - 128.0)
```

Comment:
update comment to reflect changed behavior

Reply:
We should also add this to docs